config/custom-plugin-monitor.json

```diff
@@ -0,0 +1,32 @@
+{
+  "plugin": "custom",
+  "pluginConfig": {
+    "invoke_interval": "30s",
+    "timeout": "5s",
```

Comment:
I feel like we could have global default `invoke_interval` and `timeout`, but we also need per script interval.

Reply:
For now, only `timeout` can be specified on global and per plugin. `invoke_interval` is not. Actually. When users want to do specify different `invoke_interval`, by splitting plugins running in different invoke interval into different custom plugin configs and specify them on the command line is a way to achieve the same goal.

Reply:
I'm fine with this for now, but please add TODO in code. We should have per-rule interval.

Reply:
Will do.